### ISSUE TITLE
Need Help

### ISSUE THREAD

**Issue by tacal666** [Potterli20/begonia#1]
hi, i'm sorry to open issue for this stupid question
is it possible to compile the original Mi Source ?
because lot of person say that there is some missing firmware include file
regards.

**Comment by Potterli20** [comment]
> hi, i'm sorry to open issue for this stupid question
> is it possible to compile the original Mi Source ?
> because lot of person say that there is some missing firmware include file
> regards.

Yes, the camera is missing. I haven't updated it now, because I can't find the latest camera driver. The MIUI system works, but the camera doesn't.
